worlds/metroidprime/config.py

```diff
@@ -1,1811 +1,1639 @@
-                                    "Fungal Hall A": {}
-                                },
-                                "additionalProperties": False
-                            }
-                        },
-                        "required": []
-                    },
-                    "Impact Crater": {
-                        "type": "object",
-                        "properties": {
-                            "transports": {
-                                "type": "object",
-                                "properties": {
-                                    "Crater Entry Point": {
-                                        "Artifact Temple"
-                                    },
-                                    "Essence Dead Cutscene": {
-                                        "Credits"
-                                    }
-                                },
-                                "additionalProperties": False
-                            },
-                            "rooms": {
-                                "type": "object",
-                                "properties": {
-                                    "Crater Entry Point": {},
-                                    "Crater Tunnel A": {},
-                                    "Phazon Core": {},
-                                    "Crater Missile Station": {},
-                                    "Crater Tunnel B": {},
-                                    "Phazon Infusion Chamber": {},
-                                    "Subchamber One": {},
-                                    "Subchamber Two": {},
-                                    "Subchamber Three": {},
-                                    "Subchamber Four": {},
-                                    "Subchamber Five": {},
-                                    "Metroid Prime Lair": {}
-                                },
-                                "additionalProperties": False
-                            }
-                        },
-                        "additionalProperties": False
-                    },
-                    "End Cinema": {
-                        "type": "object",
-                        "properties": {
-                            "transports": {
-                                "type": "object",
-                                "additionalProperties": False
-                            },
-                            "rooms": {
-                                "type": "object",
-                                "properties": {
-                                    "End Cinema": {}
-                                },
-                                "additionalProperties": False
-                            }
-                        },
-                        "additionalProperties": False
-                    }
-                }
-            }
-        }
-    }
-    return config
+from typing import Dict, Any
+
+
+def starting_inventory(world, item) -> bool:

```

Comment:
Changed from `self` to `world` since these functions are not methods inside of a class